projects/storefrontlib/layout/a11y/keyboard-focus/keyboard-focus.model.ts

```diff
@@ -109,10 +109,16 @@ export type TrapFocusType =
  * for the nested focusable elements. This means that the focus can not
  * "leave" the elements. If the last element is focused, the keyboard will
  * navigate to the first element and visa versa.
+ *
+ * Options:
+ * - Use the `trap` property to enable trapping for both Tab/Shift+Tab and Arrow Up/Down keys.
+ * - Use the `trapTab` property to trap only Tab/Shift+Tab, preserving the default behavior of the Arrow Up/Down keys.
  */
 export interface TrapFocusConfig extends TabFocusConfig {
-  /** traps the focus */
+  /** traps the focus for both tab/shift+tab and up/down keys */
   trap?: TrapFocusType;
+  /** traps the focus for tab/shift+tab */
+  trapTab?: TrapFocusType;
```

Comment:
You've done a nice job of introducing this property elegantly while handling any breaking change. Thanks!

My only nitpick here is to name this `trapTabOnly` instead as it distinguishes it more. However, this is opinionated.

Reply:
Thanks - I've changed to `trapTabOnly`